db/models/user.js

```diff
@@ -0,0 +1,53 @@
+var mongoose = require('mongoose');
+//add bcrypt and bluebird if not using Passport
+var bcrypt = require('bcrypt-nodejs');
+var Promise = require('bluebird');
+
+var userSchema = new mongoose.Schema({
+  name: String,
+  username: {
+    type: String,
+    required: true
+  },
+
+  //or are we using Passport?
+  password: {
+    type: String,
+    required: true
+  },
+
+  location: {
+    street: String,
+    city: String,
+    state: String,
+    //zipcode a string b/c some states have codes beginning with 0
+    zip: String
+  },
+  latitude: Number,
+  longitude: Number,
+});
```

Comment:
I think we need to store the user's "keywords" of interest, should we add it to the Users model?
I am thinking we need an array of keywords as well as the each keywords' word association (I am thinking the user should be able to remove word associations that they don't think is appropriate)